org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java

```diff
@@ -314,7 +310,7 @@ private void updateMarkers(Map<IResource, List<Issue>> issuesPerResource, Analys
           SonarLintCorePlugin.getDefault().getModulePathManager().setModulePath(moduleKey, project.getLocation().toString());
 
           String relativePath = resource.getProjectRelativePath().toString();
-          trackLocalIssues(moduleKey, resource, relativePath, rawIssues);
+          trackLocalIssues(moduleKey, relativePath, rawIssues);
           trackServerIssues(moduleKey, resource, relativePath);
         } else {
           // TODO delete if this never happens, or else handle it better
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)